app/initializers/storage-monitor.js

```diff
@@ -0,0 +1,33 @@
+import mdCalcStorage from 'mdeditor/utils/md-calc-storage';
+
+export function initialize(application) {
+
+
+  let appDefer = () => {
```

Comment:
Not sure about the utility of dismissing the message. If localstorage > 5MB the app is not going to boot and the only way to resolve it is to clear the cache(can't remember if the dev console will even work in this instance to view and recover the data).